chunky/src/java/se/llbit/chunky/block/minecraft/OpenEyeBlossom.java

```diff
@@ -0,0 +1,29 @@
+package se.llbit.chunky.block.minecraft;
+
+import se.llbit.chunky.block.MinecraftBlockTranslucent;
+import se.llbit.chunky.model.minecraft.SpriteModel;
+import se.llbit.chunky.renderer.scene.Scene;
+import se.llbit.chunky.resources.Texture;
+import se.llbit.chunky.world.Material;
+import se.llbit.chunky.world.material.TextureMaterial;
+import se.llbit.math.Ray;
+
+public class OpenEyeBlossom extends MinecraftBlockTranslucent {
```

Comment:
For compatibility with the opencl plugin, this should better be a `QuadModel` with a special `intersect` method, similar to the open eye blossom flower pot.